tests/e2e/objects/views/ArtifactsView.ts

```diff
@@ -0,0 +1,239 @@
+import type { ElectronApplication, Locator, Page } from "@playwright/test";
+import { expect } from "@playwright/test";
+import { stubDialog } from "electron-playwright-helpers";
+import path from "path";
+import { ConnectionType } from "../../connectionTypes";
+import { NotificationArea } from "../notifications/NotificationArea";
+import { Quickpick } from "../quickInputs/Quickpick";
+import { ResourcesView } from "./ResourcesView";
+import { View } from "./View";
+import { KafkaClusterItem } from "./viewItems/KafkaClusterItem";
+import { ViewItem } from "./viewItems/ViewItem";
+
+export enum SelectFlinkDatabase {
+  FromResourcesView = "Flink database action from the Resources view",
```

Comment:
Future branch: rename suggestion for when the "from compute pool" entrypoint is handled:
```suggestion
  DatabaseFromResourcesView = "Flink database action from the Resources view",
  // PoolFromResourcesView = "Flink compute pool action from the Resources view",
```